framework/deproxy_client.py

```diff
@@ -162,10 +171,9 @@ def make_requests(self, requests):
         while len(requests) > 0:
             try:
                 req = deproxy.Request(requests)
-            except:
-                tf_cfg.dbg(2, "Can't parse request")
+            except Exception as e:
+                tf_cfg.dbg(2, f"Can't parse request. Error msg: {e}")
                 req = None
-
             if req == None:
                 break
```

Comment:
You changed exception handling, now request will be sent even when `read_sized_body` raise an exception. It looks confusing, I have error message, but test completes successfully. Also `make_requests` has incorrect behaviour. On exception we break `while`, but processing of the request continues. Therefore only first request can be parsed also `valid_req_num` and others might has incorrect value. In other words we should change behaviour for such requests. Maybe add an option to `client` for ignoring parsing errors, but such option must be disabled by default. Parsing errors can help when our goal is send a valid request.

Reply:
It seems like this change only adds verbose message, broken requests sending was introduced earlier.